src/compiler/moduleNameResolver.ts

```diff
@@ -194,6 +198,15 @@ function formatExtensions(extensions: Extensions) {
     return result.join(", ");
 }
 
+function extensionsToExtensionsArray(extensions: Extensions) {

```

Comment:
Out of curiosity, how often does this happen? Is it worth caching in some way given the key is just a number?

Reply:
The number of packages AutoImportProvider will process is capped at 10, so surely not enough times to be make an observable impact.